modules/ROOT/pages/create-outbound-message-flow.adoc

```diff
@@ -19,52 +19,51 @@ A new message flow is created. +
 
 . Expand the *Source at <host-name>* section.
 . In *Endpoint:Source at <host-name>*, click *Select*.
-. To use an existing receiving endpoint: 
+. To use an existing receiving endpoint:
 .. Select the endpoint from the list and click *Select* in the lower right side of the page. +
-The endpoint is added to the message flow, and you see a message that tells you the endpoint is missing an identifier. 
+The endpoint is added to the message flow, and a message appears indicating that the endpoint is missing an identifier.
 .. Click *Add* to add a unique identifier for the endpoint.
 . To create a new receiving endpoint:
-.. Click *New* on the right of the search box. 
-.. From the *Owner* drop-down, select the endpoint's owner. 
+.. Click *New* on the right of the search box.
+.. From the *Owner* drop-down, select the endpoint's owner.
```

Comment:
```suggestion
.. From the *Owner* drop-down list, select the endpoint's owner.
```
If I were to omit any word here, it would be "drop-down" rather than "list."